mrtt-ui/src/components/ManagementStatusAndEffectiveness/ManagementStatusAndEffectivenessForm.js

```diff
@@ -170,6 +171,7 @@ const ManagementStatusAndEffectivenessForm = () => {
       <Form>
         <FormQuestionDiv>
           <StickyFormLabel>{questions.dateOfAssessment.question}</StickyFormLabel>
+          <RequiredIndicator />
```

Comment:
<img width="833" alt="image" src="https://github.com/globalmangrovewatch/gmw-users/assets/115191672/212fe187-209e-46d7-bb05-d97549b077de">

Can you nest `<RequiredIndicator />` inside StickyFormLabel so it looks like:
```
<StickyFormLabel>
  {questions.dateOfAssessment.question}
  <RequiredIndicator />
</StickyFormLabel>
```

Should make the formatting match other questions.
<img width="1005" alt="image" src="https://github.com/globalmangrovewatch/gmw-users/assets/115191672/22af8fd9-61e8-4d57-96bb-4d9d6fc21ecf">